Dockerfile

```diff
@@ -22,13 +22,21 @@ RUN \
     grep \
     iproute2 \
     iptables \
+    iptables-legacy \
     ip6tables \
     iputils \
     libcap-utils \
     libqrencode-tools \
     net-tools \
     openresolv && \
   echo "wireguard" >> /etc/modules && \
+  cd /sbin && \
+  for i in ! !-save !-restore; do \
```

Comment:
You're going to have to explain the `!` syntax here to me because I've never seen it before and can't make it work testing locally.

Reply:
Nothing special, just a placeholder character. In the for loop, I need to iterate over `iptables`, `iptables-save` and `iptables-restore`, and symlink them to `iptables-legacy`, `iptables-legacy-save` and `iptables-legacy-restore`. The common denominator is the `iptables` word, and I can iterate over the extension to it. However, the first one (iptables) does not have an extension and I can't do `for i in <blank> -save -restore; do` and instead, I add a placeholder character `!` to all three, and strip that character in the `do` part.

Without it, I could handle `iptables-save` and `iptables-legacy` in the for loop, but I would have to add separate lines for `iptables` (and `ip6tables`).

Reply:
Oh, I was testing it in bash out of habit, where it dies horribly with `bash: !-save: event not found`, forgot this is running under sh.